stress/common/src/main/java/alluxio/stress/jobservice/JobServiceBenchSummary.java

```diff
@@ -161,16 +144,6 @@ public void setStatisticsPerMethod(Map<String, SummaryStatistics> statisticsPerM
     return mStatistics.computeTimeData();
   }
 
-  private List<String> collectErrors() {
-    List<String> errors = new ArrayList<>();
-    for (Map.Entry<String, List<String>> entry : mErrors.entrySet()) {
-      // add all the errors for this node, with the node appended to prefix
-      errors.addAll(entry.getValue().stream().map(err -> entry.getKey() + ": " + err)
-          .collect(Collectors.toList()));
-    }
-    return errors;
-  }
-
```

Comment:
Now the collectErrors are renamed as collectErrorsFromAllNodes and move to [MultipleNodeBenchSummary]